### PR TITLE
add pyvista to environment

### PR DESCRIPTION
**Pull request by minrk**
needed for fenics notebook

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -4,13 +4,12 @@ channels:
 dependencies:
   - altair
   - mpich
-  - ipyparallel
+  - ipyparallel>=8.6
```
Comment (minrk) [inline environment.yml:8]: now on conda-forge, no longer need pip

**Comment by minrk** [comment]
The dolfin notebook is crashing (engine died while executing....), possibly due to a resources issue on CI.

**Comment by minrk** [comment]
@hherlyng this should add the needed dependencies. It does disable re-execution of that notebook, but we can open a new PR to remove that to figure out what else is missing.

**Review by hherlyng — COMMENTED** [review]
looks good to me! merged it now

**Pull request merged**